server/sockets/list.js

```diff
@@ -229,20 +229,48 @@ const updateList = (socket, dashboardViewClients, cohortViewClients) => {
 };
 
 const updateListHeaderState = socket => {
```

Comment:
Is this method called only once during initialization? Or on every socket event? Just wondering if `clientLocks` will be created only once.

Reply:
You're right, it was created multiple times, I had to change it